45 Rewrote: Monitor and Consume Events (#181)
casper-network/docs-new: source/docs/casper/developers/dapps/monitor-and-consume-events.md

## Listen to the Event Stream

Setup an event listener in your app using the following:

<Tabs>

<TabItem value="js" label="JavaScript">

```javascript
const { EventStream, EventName } = require("casper-js-sdk")

const es = new EventStream("http://NODE_ADDRESS:9999/events/" + CHANNEL)
es.start()
es.subscribe(EventName.EVENT_NAME, eventHandler)

function eventHandler(event) {
    console.log(event)
}
```

</TabItem>

<TabItem value="python" label="Python">

```python
from pycspr import NodeClient, NodeConnection, NodeEventChannel, NodeEventType

def eventHandler(event):
    print(event)

client = NodeClient(NodeConnection(host = "NODE_ADDRESS", port_rpc = 7777))
client.get_events(eventHandler, NodeEventChannel.CHANNEL, NodeEventType.EVENT_NAME)
```

</TabItem>

</Tabs>

*Note: You can find active online peers' `NODE_ADDRESS`es at [cspr.live](https://cspr.live/tools/peers) for mainnet and [testnet.cspr.live](https://testnet.cspr.live/tools/peers) for testnet.*

> Where `CHANNEL` is `main` for an event type of `ApiVersion`, `BlockAdded`, `DeployExpired`, `DeployProcessed`, `Fault` or `Step`. `CHANNEL` is `deploys` for `DeployAccepted`, and `CHANNEL` is `sigs` for `FinalitySignature`.

Reviewer: Can you rephrase/restructure this? I think I know what is meant, but if you read that back to yourself I think you'll agree that it's as clear as mud on first reading. 
```suggestion
Where `CHANNEL` is set to;
 `main` for the following event types;
 `ApiVersion`, `BlockAdded`, `DeployExpired`, `DeployProcessed`, `Fault` or `Step`. 
`deploys` for event types;
 `DeployAccepted`, or 
`sigs` for event type `FinalitySignature`.
```
